sdk/metricsadvisor/ai-metrics-advisor/test/public/datafeed.spec.ts

```diff
@@ -23,791 +20,787 @@ import { createRecordedAdminClient, testEnv } from "./util/recordedClients";
 import { Recorder } from "@azure/test-utils-recorder";
 
 describe("MetricsAdvisorAdministrationClient datafeed", () => {
-    let client: MetricsAdvisorAdministrationClient;
-    let recorder: Recorder;
-    let feedName: string;
-    let appInsightsFeedName: string;
-    let sqlServerFeedName: string;
-    let cosmosFeedName: string;
-    let dataExplorerFeedName: string;
-    let azureTableFeedName: string;
-    let httpRequestFeedName: string;
-    let influxDbFeedName: string;
-    let mongoDbFeedName: string;
-    let mySqlFeedName: string;
-    let postgreSqlFeedName: string;
-
-    const apiKey = new MetricsAdvisorKeyCredential(
-      testEnv.METRICS_ADVISOR_SUBSCRIPTION_KEY,
-      testEnv.METRICS_ADVISOR_API_KEY
-    );
-
-    beforeEach(function() {
-      // eslint-disable-next-line no-invalid-this
-      ({ recorder, client } = createRecordedAdminClient(this, apiKey));
-      if (recorder && !feedName) {
-        feedName = recorder.getUniqueName("js-test-datafeed-");
-      }
-      if (recorder && !appInsightsFeedName) {
-        appInsightsFeedName = recorder.getUniqueName("js-test-appInsightsFeed-");
-      }
-      if (recorder && !sqlServerFeedName) {
-        sqlServerFeedName = recorder.getUniqueName("js-test-sqlServerFeed-");
-      }
-      if (recorder && !cosmosFeedName) {
-        cosmosFeedName = recorder.getUniqueName("js-test-cosmosFeed-");
-      }
-      if (recorder && !dataExplorerFeedName) {
-        dataExplorerFeedName = recorder.getUniqueName("js-test-dataExplorerFeed-");
-      }
-      if (recorder && !azureTableFeedName) {
-        azureTableFeedName = recorder.getUniqueName("js-test-tableFeed-");
-      }
-      if (recorder && !httpRequestFeedName) {
-        httpRequestFeedName = recorder.getUniqueName("js-test-httpRequestFeed-");
-      }
-      if (recorder && !influxDbFeedName) {
-        influxDbFeedName = recorder.getUniqueName("js-test-influxdbFeed-");
-      }
-      if (recorder && !mongoDbFeedName) {
-        mongoDbFeedName = recorder.getUniqueName("js-test-mongoDbFeed-");
+  let client: MetricsAdvisorAdministrationClient;
+  let recorder: Recorder;
+  let feedName: string;
+  let appInsightsFeedName: string;
+  let sqlServerFeedName: string;
+  let cosmosFeedName: string;
+  let dataExplorerFeedName: string;
+  let azureTableFeedName: string;
+  let httpRequestFeedName: string;
+  let influxDbFeedName: string;
+  let mongoDbFeedName: string;
+  let mySqlFeedName: string;
+  let postgreSqlFeedName: string;
+
+  const apiKey = new MetricsAdvisorKeyCredential(
+    testEnv.METRICS_ADVISOR_SUBSCRIPTION_KEY,
+    testEnv.METRICS_ADVISOR_API_KEY
+  );
+
+  beforeEach(function() {
+    // eslint-disable-next-line no-invalid-this
+    ({ recorder, client } = createRecordedAdminClient(this, apiKey));
+    if (recorder && !feedName) {
```

Comment:
```suggestion
    if (recorder && !feedName) {
```
Why recorder here?

Reply:
Not 100% sure, let's ask @KarishmaGhiya and @jeremymeng .